special-interest-groups/sig-planner/roles-and-organization-management.md

```diff
@@ -0,0 +1,64 @@
+# Planner SIG Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Planner SIG is to have
+the **Active Contributor** community role in the planner/statistics package.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 3 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Planner SIG Reviewers, Committer or Maintainer.
+
+#### From `Active Contributor` to `Reviewer`
+
+Requirements:
+
+* Have at least 10 PRs merged in the planner/statistics package within one year.
```

Comment:
How about 20?

Reply:
One more requirement about the number of valuable issues too.